### ISSUE TITLE
Patch for r2368

### ISSUE THREAD

**Issue by svigerske** [coin-or/Ipopt#218]
Issue created by migration from Trac.

Original creator: @ghackebeil

Original creation time: 2013-09-02 23:16:02

Assignee: ipopt-team

Debug build is failing...

```
Index: Interfaces/IpStdInterfaceTNLP.cpp
===================================================================
--- Interfaces/IpStdInterfaceTNLP.cpp	(revision 2370)
+++ Interfaces/IpStdInterfaceTNLP.cpp	(working copy)
`@``@` -387,7 +387,7 `@``@`
         non_const_x_[i] = x[i];
       }
     }
-    DBG_ASSERT(non_const_x && "non_const_x is NULL after apply_new_x");
+    DBG_ASSERT(non_const_x_ && "non_const_x is NULL after apply_new_x");
   }
 
 } // namespace Ipopt
```

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-09-03 09:39:48

Thanks, fixed with db08ac8965.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-09-05 13:44:11

Resolution: fixed
